src/test/msi/TestData/UIExtensionTests/LocalizedWixUI/product.wxs

```diff
@@ -0,0 +1,13 @@
+<!-- Copyright (c) .NET Foundation and contributors. All rights reserved. Licensed under the Microsoft Reciprocal License. See LICENSE.TXT file in the project root for full license information. -->
```

Comment:
Can we name this `LocalizedWixUI.wxs`?

I've stopped using `Product.wxs` and `Package.wxs` in my tests because when there are so many with the same name. The generic name makes it hard to figure out what value the .wxs file provides and which ones belong to which test (the directory name is _required_).